### ISSUE TITLE
Epic: Flash engine-integration plan (breadth tier of one simulation engine)

### ISSUE THREAD

**Issue by realityinspector** [timepointai/timepoint-flash#73]
Tracker for the Flash engine-integration plan (plan PR #65, branch `docs-integration-timepoint-flash-jun13`, specs in `docs/execution-prs/`).

**Positioning:** Flash is the **breadth (quick-sim) tier** of the **one simulation engine** it forms with GSS/Timepoint Pro (depth tier), in the unified grounded-world-model loop. The unifying discipline: **grounded confidence over fabricated confidence** — a quick-sim score must be comparable/calibrated, fail-closed, grounded, slug-guarded, cost-bounded.

## Build order (calibration-spine-first, mirrors the GSS DAG)

**Root — make the score honest**
- [ ] #66 — PR-02: Fail-closed scoring + per-metric confidence (no fabricated 0.5) *(root, no deps)*

**Keystone — make it falsifiable + comparable**
- [ ] #67 — PR-01: Quick-sim calibration spine (held-out ground truth) **KEYSTONE** — *cross-repo → `timepoint-snag-bench` (director opens that PR)* — after #66

**Ground it / guard it (parallelizable, run guard early)**
- [ ] #68 — PR-03: Quick-sim grounding pass (liveness + temporal + Clockchain anchor; Spine-E bridge) — after #66
- [ ] #69 — PR-04: Model-slug liveness guard (fail loud; mirrors GSS PR-00c) — *independent, early*
- [ ] #72 — PR-07: Rank-stability CIs (tied shortlist reads as tied; mirrors GSS PR-01b) — after #66

**Seam — make breadth→depth a real contract**
- [ ] #70 — PR-05: Flash→Pro calibrated handoff (one scale) — *cross-repo → web-app + pro-cloud (director opens those PRs)* — after #67, #66

**Keep it honest over time**
- [ ] #71 — PR-06: Score flight-recorder + calibration-drift canary (mirrors GSS PR-XX-obs / Clockchain data-canary) — after #67

## DAG
```
PR-02 (#66) ──┬─> PR-01 (#67, KEYSTONE) ──┬─> PR-05 (#70, Flash->Pro handoff)
              │                            └─> PR-06 (#71, drift canary)
              ├─> PR-03 (#68, grounding)
              └─> PR-07 (#72, rank CIs)
PR-04 (#69, slug guard)  — independent, early
```

## Gates / realities
- **Cross-repo** (#67 snag-bench, #70 web-app + pro-cloud): branches pushed in those repos but the director opens those PRs manually (`feedback_cross_repo_pr_opening`).
- **LOOP-layer training loop is gated** (do not build) — same gate as GSS PR-03; PR-01 stops at calibration, not RLSF.
- **Deploy-private**: live `flash.timepointai.com` runs from `timepoint-flash-deploy-private-feb-2026` (surgical cherry-pick sync, not merge); these specs land in the open-source repo. Auth is gateway-owned (`AUTH_ENABLED=false` + `X-Gateway-Metered`). Quick-sim is **Stability-key-neutral** (no inline image render).
